.bcr/metadata.template.json

```diff
@@ -0,0 +1,13 @@
+{
+  "homepage": "https://github.com/bazel-contrib/rules_cuda",
+  "maintainers": [
+    {
+      "email": "james.sharpe@zenotech.com",
+      "github": "jsharpe",
+      "name": "James Sharpe"
+    }
```

Comment:
@cloudhan @ryanleary note that this is the maintainer of the BCR entry not necessarily the repo; feel free to add yourself to this list if you want but I'm happy to be listed here.